frontend/src/pages/data-product-settings/components/data-product-settings-table/data-product-settings-table-columns.tsx

```diff
@@ -0,0 +1,138 @@
+import { Button, Flex, Popconfirm } from 'antd';
+import { TFunction } from 'i18next';
```

Comment:
You can change this to a `type` import to get full erasure.
(https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html)

```suggestion
import type { TFunction } from 'i18next';
```